examples/bridge-app/bridge-common/bridge-app.matter

```diff
@@ -1638,6 +1642,10 @@ endpoint 0 {
   }
 
   server cluster GeneralDiagnostics {
+    emits event HardwareFaultChange;
```

Comment:
Only `BootReason` event is implemented for bridge app.

Reply:
Is this reflected in the zap file? Could you point me to it?

I would like to make sure that whatever matter files reflect is what our zap config says ... zap may be wrong and then matter files are for sure easier to review.

Reply:
This was based on an old master. Synced with master and events were updated.